doc/src/content/atoms/checkbox/props.md

```diff
@@ -0,0 +1,17 @@
+---
+path: "/checkbox/props"
+component: "checkbox"
+type: "atom"
+title: "Checkbox"
+---
+
+Use `Checkbox` component to render `input` with type `checkbox`.
+
+Wrap the checkbox with styled components to change the styles.
+
+| Prop | Description | Type | Default |
+| ------ | ----------- | ---- | ------- |
+| name | name for the input field | string | null |
+| id | id for the input field | string | null |
+| checked | initial value of the checkbox | boolean | false |
```

Comment:
checked and disabled can have better description like "Whether checkbox is checked by default" or "disabled by default"

Reply:
Agreed.